src/schema/v2/order/types/DisplayTexts.ts

```diff
@@ -42,71 +43,123 @@ export const DisplayTexts: GraphQLFieldConfig<OrderJSON, ResolverContext> = {
   resolve: (order) => resolveDisplayTexts(order),
 }
 
-const formatMessage = (parts: string[]) => parts.join("<br/><br/>")
+const formatMessage = (parts: string[], joinWith = "<br/><br/>") =>
+  parts.join(joinWith)
+
 const resolveDisplayTexts = (order: OrderJSON) => {
   const isPickup = order.fulfillment_type == "pickup"
+  const stateExpireTime =
+    order.buyer_state_expires_at && moment.tz(order.buyer_state_expires_at)
+  const formattedStateExpireTime =
+    stateExpireTime &&
+    `${stateExpireTime.format("MMM D")}, ${stateExpireTime.format("h:mma z")}`
```

Comment:
Do want to display in local time zone or the fixed America/New_York time zone? There is this [helper](https://github.com/artsy/metaphysics/blob/07be03ae492c0a87ddf97289474acfe295425bf2/src/lib/date.ts#L55) that can be reused, with slightly different format.

Reply:
I looked at that one and was not sure if we want to reuse ir. We can follow up with exact formatting adjustments. If you are ok with general direction here - I'm inclined to merge it for now and have this going all the way to UI. So we can start sharing this with Eigen.

The good thing - adjustment will be just here if all works as intended :)